steps/xcode-test-bundle-builder/0.0.5/step.yml

```diff
@@ -0,0 +1,81 @@
+title: Build xcode test runner for remote testing service
+summary: Build your XCTest or XCUITest runner to upload to remote testing service
+description: Build your XCTest or XCUITest bundle to upload to remote testing service
+  such as AWS Device Farm
+website: https://github.com/otusweb/xCode-test-bundle-builder-for-bitrise
+source_code_url: https://github.com/otusweb/xCode-test-bundle-builder-for-bitrise
+support_url: https://github.com/otusweb/xCode-test-bundle-builder-for-bitrise/issues
+published_at: 2019-02-19T20:45:09.554567+01:00
+source:
+  git: https://github.com/otusweb/xCode-test-bundle-builder-for-bitrise.git
+  commit: 71891e271dd1f93614dd65ff366b752862265faf
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
+- build
+- xcode
```

Comment:
Please remove the `xcode` type tag.
Valid type tags: 
https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-grouping-convention